Source/WTF/wtf/Condition.h

```diff
@@ -142,6 +142,12 @@ struct ConditionBase {
         return waitForSecondsImpl(lock, absoluteTimeoutSeconds - monotonicallyIncreasingTime());
     }
     
+    template<typename LockType>
+    bool waitForSeconds(LockType& lock, double relativeTimeoutSeconds)
```

Comment:
This is needed because we cannot let the qtdemux select the decryptor without us checking with the JS app and therefore it needs to be done here and wait for the result. Otherwise, when we got the answer, we might have the wrong decryptor selected or even worse, if we just let the decryptors filter the events, we might be screwed cause we wouldn't be getting all of them, which is needed for 0.1b.